stock_inventory_revaluation/wizards/stock_inventory_revaluation_mass_post.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# © 2015 Eficent Business and IT Consulting Services S.L.
+# - Jordi Ballester Alomar
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, models, _, exceptions
+
+
+class StockInventoryRevaluationMassPost(models.TransientModel):
+
+    _name = 'stock.inventory.revaluation.mass.post'
+    _description = 'Post multiple inventory revaluations'
+
+    @api.model
+    def default_get(self, fields):
+        res = super(StockInventoryRevaluationMassPost, self).default_get(
+            fields)
+        revaluation_ids = self.env.context['active_ids'] or []
```

Comment:
I'm assuming this is the line that allow multiple revaluation. Don't you need to return it on res? Beside you already has it here https://github.com/OCA/stock-logistics-warehouse/pull/121/files#diff-42c033da95b8069ee3b5c33be0ce91e0R29.